core/extension/SCsub

```diff
@@ -2,6 +2,12 @@
 
 Import("env")
 
+import make_wrappers
+from platform_methods import run_in_subprocess
+
+env.CommandNoCache(["ext_wrappers.gen.inc"], "make_wrappers.py", run_in_subprocess(make_wrappers.run))
```

Comment:
Since we clone the env just below, this should likely run in `env_extension` (probably doesn't make much difference here but just for consistency).

Reply:
I see it's done the same way in `core/object/SCsub`. I don't think those need a cloned env in the first place, but well maybe it's not so important for this PR.